doc/release/README.multilocale

```diff
@@ -88,10 +89,19 @@ To run Chapel on multiple locales, the following steps are required:
            to use normal ssh-agent/ssh-add capabilities to support
            password-less ssh-ing.
 
-           Note that it is possible to configure GASNet to launch jobs
-           with SLURM using the following command:
+           Also note that if you are using SSH to launch jobs, you might get a
+           login banner printed out along with your program's output. We have
+           found the following setting useful to disable such printing:
+
+             export GASNET_SSH_OPTIONS="-o LogLevel=Error"
```

Comment:
Yeah, the message boards I was reading seemed to indicate that would *not* display certain errors though, like e.g. Connection Reset.